### PR TITLE
User interface improvements

### PR DESCRIPTION
**Pull request by szok**
- Add add record validation from API
- Add wrap text to records list
- Add first, last page to pagination component
- Change styles in Domain header page
- Change redirect location after record/record request save
- Add modified date to record list
- Add RecordRequest new/old value view

zrzut ekranu 2016-06-14 o 12 40 22
zrzut ekranu 2016-06-14 o 12 40 28
zrzut ekranu 2016-06-14 o 14 32 01
zrzut ekranu 2016-06-14 o 14 37 37

### CODE REVIEW

File: ui/static/app/record-request/record-request-detail.component.ts [hunk 1]
```diff
@@ -12,14 +12,18 @@ import { RecordRequest } from "./record-request";
 @Component({
   templateUrl: "/static/app/templates/record-request-detail.component.html",
   providers: [DomainService, RecordService, RecordRequestService],
-  styles: ["td span { cursor:pointer; }"]
+  styles: [`
+    td span { cursor:pointer; }
+    :host >>> .new { color: green; }
+    :host >>> .old { color: red; }
```
Comment (ghost) [inline ui/static/app/record-request/record-request-detail.component.ts:18]: what's this `>>>`?

Comment (szok) [inline ui/static/app/record-request/record-request-detail.component.ts:18]: Adds CSS for the Host (parent component)
It uses this style to work in InnerHTML

File: ui/static/app/templates/domain.component.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="page-header">
   <h1>Domains
-    <small>You can find who owns a given domain here. You can not edit anything here.</small>
+    <small>You can find who owns a given domain. You can not edit anything here.</small>
```
Comment (ghost) [inline ui/static/app/templates/domain.component.html]: `can not` -> `cannot`?

File: ui/static/app/record-request/record-request-detail.component.ts (hunk 1 above)
Comment (xliiv) [inline ui/static/app/record-request/record-request-detail.component.ts]: Then this is weird Oo

File: ui/static/app/templates/domain.component.html (hunk 2 above)
Comment (szok) [inline ui/static/app/templates/domain.component.html]: Done

**Comment by ghost** [comment]
:ok:

**Pull request merged**